package.json

```
@@ -7,10 +7,12 @@
     "dev": "nuxt dev",
     "generate": "nuxt generate",
     "preview": "nuxt preview",
-    "postinstall": "nuxt prepare"
+    "postinstall": "nuxt prepare",
+    "git:commit": "git add . && git-cz"
```

```suggestion
    "git:commit": "git add -A && git-cz"
```
so deleted files are also included?

---

Deleted files are also staged po with 'git add .' I think. But yes po, I think -A flag is much better